<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>690</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Contact Book</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(210, 255, 252);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="searchWidget" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>451</width>
      <height>71</height>
     </rect>
    </property>
    <widget class="QLineEdit" name="searchLine">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>321</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Franklin Gothic Book</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border: 2px solid grey;
border-radius: 10px;</string>
     </property>
     <property name="placeholderText">
      <string>  Search contacts by name...</string>
     </property>
    </widget>
    <widget class="QLabel" name="searchImage">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>0</y>
       <width>71</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>search_icon_small.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>570</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Franklin Gothic Book</family>
      <pointsize>48</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: lightgreen;
border-radius: 35px;
color: white;
</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>451</width>
      <height>431</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaContacts">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>449</width>
       <height>429</height>
      </rect>
     </property>
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>411</width>
        <height>61</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="noContactsLabel">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>200</y>
        <width>321</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Franklin Gothic Book</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>There is no contacts so far...</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>490</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;</string>
   </property>
   <widget class="QMenu" name="menuOpenfrom_file">
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Open from file</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSave">
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Save</string>
    </property>
   </widget>
   <addaction name="menuOpenfrom_file"/>
   <addaction name="menuSave"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
